src/randomizerData.tsx

```diff
@@ -49,7 +49,5 @@ export const classes: string[] = [
     'Rogue',
     'Sorcerer',
     'Warlock',
-    'Wizard',
-    'Artificer',
-    'Blood Hunter'
+    'Wizard'
```

Comment:
I guess I can live with not being an artificer :(

Reply:
I feel you... I'm a Blood Hunter in my campaign and love it! Maybe we can make a pay-wall for the users! Muahaha!